bin/generate-index

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const { docTitle } = require('../constants');
+const chapterHelper = require('../src/helper/chapterHelper');
+const { readFileSync, writeFile } = require('fs');
+const { resolve } = require('path');
+const jsyaml = require('js-yaml');
+const fileProcessor = require('./fileProcessor');
+
+const loadedOutline = jsyaml.safeLoad(readFileSync(`${__dirname}/../config/outline.yaml`, 'utf8'));
+
+const index = `# ${docTitle}\n\n${loadedOutline.map((chapter) => chapterHelper.processChapter(chapter, fileProcessor)).join('\n\n')}`;
+
+writeFile(resolve(__dirname, '../src/pages/docs/index.md'), index, error => error && console.warn(error));
```

Comment:
Is it really necessary to generate a Markdown file? Can't we use directly the existing YAML file as a source for Gatsby?

https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-yaml
https://github.com/gatsbyjs/gatsby/issues/3195